tonic_validate/validate_scorer.py

```diff
@@ -70,17 +71,24 @@ def __init__(
         self.max_parsing_retries = max_parsing_retries
         self.max_llm_retries = max_llm_retries
         self.fail_on_error = fail_on_error
-        logger.setLevel(logging.ERROR if quiet else logging.INFO)
         self.quiet = quiet
         self.telemetry = Telemetry()
+        logger.setLevel(logging.ERROR if quiet else logging.INFO)
+
         try:
             self.encoder = tiktoken.encoding_for_model(model_evaluator)
         except Exception as _:
             logger.info("Defaulting to cl100k_base for measuring token count")
             self.encoder = tiktoken.get_encoding("cl100k_base")
-        self.openai_service = OpenAIService(
-            self.encoder, self.model_evaluator, max_retries=self.max_llm_retries
-        )
+
+        if 'gpt' in self.model_evaluator.lower():
```

Comment:
This is not backwards compatible. On azure, the gpt models have custom names (e.g. you could name a gpt-4 instance something like "llm"). So if someone is using azure, it's very likely their model name doesn't contain GPT in it which will break it when they upgrade